docs/source/hardware_and_software_configuration/configuring/configuring_uvc_camera/configuring-uvc-camera.rst

```diff
@@ -56,11 +56,13 @@ Note that the Modern Robotics Core Power Distribution Module cannot be
 used for this task since its data transfer speed is not fast enough to
 work with the USB-connected webcam.
 
-Also note that rule c(iii) permits the use of a powered USB hub to make
-this connection. If a team uses a powered USB hub, the power to operate
-the USB hub can only come from either of the following sources:
+Also note that the Competition Manual permits the use of a powered USB
+hub to make this connection. See rule R617 for the full details. If a
+team uses a powered USB hub, the power to operate the USB hub can only
+come from either of the following sources:
 
-1. An externally connected USB 5V Battery Pack.
+1. An externally connected COTS USB Battery Pack in compliance with the
+   Competition Manual. See rule R602 for the full details.
```

Comment:
Rephrase to not point to a specific rule number. Rule numbers may shift year to year.  

Suggest something like:

> Check the Robot Construction rules in the [Competition Manual](https://ftc-resources.firstinspires.org/file/ftc/game/manual) for details.